pyproject.toml

```diff
@@ -2,6 +2,34 @@
 requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]~=6.0"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "diepvries"
+version = "0.9.2"
```

Comment:
This should be a dynamic attribute, as the version is computed with git tags and `setuptools_scm`, right?

Reply:
Yes, definitely. I also bumped the minimum setuptools and setuptools_scm version as per https://github.com/pypa/setuptools_scm?tab=readme-ov-file#pyprojecttoml-usage